editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/navigator/CamelCtxNavContentProvider.java

```diff
@@ -47,17 +47,13 @@
 	@Override
 	public Object[] getChildren(Object parent) { 
 		if(parent instanceof IFile){	
-			return getRoutes((IFile)parent,true);
+			return getRoutes((IFile)parent, false); // FUSETOOLS-2363 -> when doing lazy loading routes will duplicate
```

Comment:
so it means that the lazy loading is not used anymore?
What do you think to remove the dead code? We can reference the commit which remove it in the JIRA to improve it later on.